src/game/Spell.cpp

```diff
@@ -2982,9 +2982,12 @@ void Spell::cast(bool skipCheck)
     // CAST SPELL
     SendSpellCooldown();
 
-    TakePower();
-    TakeReagents();                                         // we must remove reagents before HandleEffects to allow place crafted item in same slot
-    TakeAmmo();
+    if (!m_IsTriggeredSpell)
```

Comment:
But would that mean that the rest of the changes are still good and it's only the check that's invalid, or the whole patch that's invalid? (I havn't looked into this in detail myself, I just saw the open issue and was going through a repo, yours as it seems, and saw a commit which was supposed to fix the issue and it seemed somewhat legit so I thought I would PR it)